pkg/module/kubebuilder/project.go

```diff
@@ -155,13 +156,23 @@ func (p *Project) DefaultCR(s step.Step) ([]byte, error) {
 	defaultCR := ""
 	if len(d) > 1 {
 		// ask for specific file
-		names := []string{}
-		for _, f := range d {
-			names = append(names, f.Name())
+		var promptString strings.Builder
+		promptString.WriteString(fmt.Sprintf("Please specify the file to use as default CR in %s:\n", samplesDir))
+
+		files := map[int32]string{}
```

Comment:
please not use int32, just int, the golang compiler will decide which during compile time based on your OS, if 64 bit then int64 or int32 in 32 bit system.

Reply:
That's an helpful insight :)